plugins/ImageGraph/API.php

```diff
@@ -238,8 +238,11 @@ public function get(
 
             $ordinateColumns = array();
             if (empty($columns)) {
-                $ordinateColumns[] =
-                    empty($reportColumns[self::DEFAULT_ORDINATE_METRIC]) ? key($metadata['metrics']) : self::DEFAULT_ORDINATE_METRIC;
```

Comment:
isn't `key($metadata['metrics'])` just `'metrics'` every time? Sorry if I misunderstand this line of code...

Reply:
@justinvelluppillai I remember the error was the key 'metrics' does not exist in $metadata, then on PHP 8.0, it throws an error call key on null when `empty($reportColumns[self::DEFAULT_ORDINATE_METRIC])` is empty.